### PR TITLE
bugfix/COR-1239-adjust-choropleth-tooltip-width

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard28**
# Summary

This ticket is regarding an issue with Choropleth tooltips. Certain safety regions and municipalities have names which are longer than others, thereby causing the tooltip for that area to be wider than names which are shorter. This created a visual inconsistency which has been fixed in this PR along with some minor refactoring.

Note: This PR addresses the issue in COR-1239, however the fix also covers the issue described in COR-1276. 

### Before - Regular Choropleth, Municipality
image

### After - Regular Choropleth, Municipality
image

### Before - Regular Choropleth, Safety Region
image

### After - Regular Choropleth, Safety Region
image

### Before - Municipality selection Choropleth
image

### After - Municipality selection Choropleth
image

### Before - Safety region selection Choropleth
image

### After - Safety region selection Choropleth
image

### CODE REVIEW

File: packages/app/src/style/preset.ts [hunk 1]
```diff
@@ -102,17 +104,17 @@ export const nestedHtml = {
...
+  ol: { ml: space[4] },
   a: { textDecoration: 'underline' },
 
   /**
    * Apply some special margin styles to "stick" headings to their content.
    */
   'h1, h2, h3, h4, h5, h6': {
-    mt: 4,
+    mt: space[4],
     mb: -2,
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/style/preset.ts:118]: @VWSCoronaDashboard26 I'm not sure how to handle `-2` 🤔

Comment (VWSCoronaDashboard26) [reply packages/app/src/style/preset.ts:118]: I believe you could use `-${space[2]}`. At least, that's how I've previously used it in styled components.

File: packages/app/src/components/choropleth/tooltips/tooltip-content.tsx [hunk 2]
```diff
@@ -19,12 +19,8 @@ export const TooltipContent = ({ title, link, children }: TooltipContentProps) =
   return (
     <StyledTooltipContent as={link ? 'a' : 'div'} href={link ? link : undefined} isInteractive={isTouch} aria-live="polite">
       <StyledTooltipHeader>
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/components/choropleth/tooltips/tooltip-content.tsx:21]: Maybe removed 'styled' from this TooltipHeader component?

**Review by VWSCoronaDashboard21 — APPROVED** [review]
Looks good to me.

File: packages/app/src/components/choropleth/tooltips/tooltip-content.tsx [hunk 3]
```diff
@@ -19,12 +19,8 @@ export const TooltipContent = ({ title, link, children }: TooltipContentProps) =
...
-        <Text variant="choroplethTooltipHeader">
-          <StyledLocationIcon>
-            <Location />
-          </StyledLocationIcon>
-          {title}
-        </Text>
+        <LocationIcon />
+        <Text variant="choroplethTooltipHeader">{title}</Text>
         {link && <StyledChevronRight />}
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/components/choropleth/tooltips/tooltip-content.tsx:24]: Maybe remove 'Styled' from the ChevronRight component.

File: packages/app/src/components/choropleth/tooltips/tooltip-content.tsx (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/choropleth/tooltips/tooltip-content.tsx:21]: This is not yet discussed in the Frontend Chapter. So for now I'll close this comment.

File: packages/app/src/components/choropleth/tooltips/tooltip-content.tsx (hunk 3 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/choropleth/tooltips/tooltip-content.tsx:24]: This is not yet discussed in the Frontend Chapter. So for now I'll close this comment.

File: packages/app/src/style/preset.ts [hunk 4]
```diff
@@ -102,18 +104,18 @@ export const nestedHtml = {
...
   a: { textDecoration: 'underline' },
 
   /**
    * Apply some special margin styles to "stick" headings to their content.
    */
   'h1, h2, h3, h4, h5, h6': {
-    mt: 4,
-    mb: -2,
+    mt: space[4],
+    mb: -space[2],
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/style/preset.ts:118]: Have you tried renaming this to `marginTop` and `marginBottom`? Or even combining these two into a single.
```
margin: `${space[4]} -${space[2]}`

**Review by VWSCoronaDashboard26 — APPROVED** [review]
Minor comment, but pre-approved.

File: packages/app/src/style/preset.ts (hunk 4 above)
Comment (VWSCoronaDashboard28) [reply packages/app/src/style/preset.ts]: Good one

**Pull request merged**